src/commands/cctv.js

```diff
@@ -0,0 +1,36 @@
+const Builder = require('@discordjs/builders');
+
+const DiscordMessages = require('../discordTools/discordMessages.js');
+
+module.exports = {
+	name: 'cctv',
+
+	getData(client, guildId) {
+		return new Builder.SlashCommandBuilder()
+			.setName('cctv')
+			.setDescription(client.intlGet(guildId, 'commandsCctvDesc'))
+            .addStringOption(option =>
+                option.setName('monument')
+                    .setDescription('Rust Monument')
+                    .setRequired(true)
+                    .addChoices(
+                        { name: 'Abandoned Military Base', value: 'Abandoned Military Base' },
```

Comment:
I think there are translations for the different monuments already so use them instead of just using the English one.